.github/workflows/publish.yml

```
@@ -0,0 +1,24 @@
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 16
+          registry-url: https://npm.pkg.github.com/
+      - name: Install dependencies and build 🔧
+        run: npm ci && npm run build
+      - name: Publish package on Github 📦
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}} 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)